graphics/osxphotos/Portfile

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               python 1.0
+
+name                    osxphotos
+version                 0.63.0
+revision                0
+
+categories              graphics
+
+supported_archs         noarch
```

Comment:
should `platforms darwin any` be added here? Given the download file name on PyPI (`
osxphotos-0.63.4-py3-none-any.whl `) that suggests to be the case from the `none-any` part.